### PR TITLE
test: custom content in local http server

### PR DESCRIPTION
**Pull request by sadym-chromium**
* Add possibility to provide a URL with a custom content.
* Remove `data:text/html` fixture
* Address concurrency issue in local http server by adding several instances.

### CODE REVIEW

File: tests/log/test_log_entry_added.py [hunk 1]
```diff
@@ -353,10 +354,12 @@ async def test_exceptionThrown_logEntryAddedEventEmitted(
...
             "stackTrace": {
                 "callFrames": [{
-                    "url": "",
+                    "url": url,
                     "functionName": "",
                     "lineNumber": 0,
-                    "columnNumber": 14
+                    # Column number is a magical constant. It depends on the
+                    # html fixture wrapping content in document tag.
+                    "columnNumber": 127
```
Comment (Lightning00Blade) [inline tests/log/test_log_entry_added.py:362]: How about we use `ANY_NUMBER`?

File: tests/browsing_context/test_traverse_history.py [hunk 2]
```diff
@@ -21,28 +21,32 @@
 
 
 @pytest.mark.asyncio
-async def test_traverse_history(websocket, context_id, html):
+async def test_traverse_history(websocket, context_id):
+    urls = []
     for i in range(HISTORY_LENGTH + 1):
-        await goto_url(websocket, context_id, html(i))
+        # TODO: use `html` fixture instead.
```
Comment (Lightning00Blade) [inline tests/browsing_context/test_traverse_history.py:27]: Will you address it now or in another PR?

**Review by Lightning00Blade — APPROVED** [review]
LGTM

File: tests/log/test_log_entry_added.py (hunk 1 above)
Comment (sadym-chromium) [reply tests/log/test_log_entry_added.py:362]: This is expected to be a constant, so let's keep it

File: tests/browsing_context/test_traverse_history.py (hunk 2 above)
Comment (sadym-chromium) [reply tests/browsing_context/test_traverse_history.py:27]: I'll file an issue

**Pull request merged**